pom.xml

```diff
@@ -2932,7 +2932,7 @@
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>3.0.0-M5</version>
+          <version>3.0.0-M7</version>
```

Comment:
@panbingkun There is a behavioral difference between `3.0.0-M5` and `3.0.0-M7`:

`mvn clean install -pl core -Dtest=none` pass with 3.0.0-M5`, but failed with `3.0.0-M7` as follows:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M7:test (default-test) on project spark-core_2.12: No tests matching pattern "none" were executed! (Set -Dsurefire.failIfNoSpecifiedTests=false to ignore this error.) -> [Help 1]
```
seems that configuration `<failIfNoSpecifiedTests>false</failIfNoSpecifiedTests>` needs to be added

Reply:
OK